### PR TITLE
corrects the branch name to pull the code from

### PR DESCRIPTION
**Pull request by rkothiya**
fixes: #113

Signed-off-by: Rinku Kothiya <rkothiya@redhat.com>

### CODE REVIEW

File: jobs/nightly-rpm-builds.yml [hunk 1]
```diff
@@ -49,17 +58,17 @@
...
-            CENTOS_VERSION=7
+            CENTOS_VERSION=8
 
             CENTOS_ARCH=x86_64
     - trigger-builds:
         - project: gluster_build-rpms
           block: false
           predefined-parameters:
-            GERRIT_BRANCH=release-6
+            GERRIT_BRANCH=release-7
```
Comment (deepshikhaaa) [inline jobs/nightly-rpm-builds.yml:70]: Any specific reason to not build release-7 branch on other centos versions?

File: jobs/nightly-rpm-builds.yml [hunk 2]
```diff
@@ -13,7 +13,16 @@
         - project: gluster_build-rpms
           block: false
           predefined-parameters:
-            GERRIT_BRANCH=master
+            GERRIT_BRANCH=devel
```
Comment (deepshikhaaa) [inline jobs/nightly-rpm-builds.yml:16]: Can you please fix the variable name as GITHUB_BRANCH?

File: jobs/nightly-rpm-builds.yml (hunk 1 above)
Comment (rkothiya) [reply jobs/nightly-rpm-builds.yml]: I was not sure if we have enough build machines to do that. Now Included other centos versions for building release-7

**Review by nixpanic — CHANGES_REQUESTED** [review]
You are missing the updates to the job that contains the parameters. I do not think this will work without it. Please see https://github.com/gluster/centosci/blob/master/jobs/build-rpms.yml

You might also need to update GERRIT_BRANCH -> GITHUB_BRANCH and master -> devel in other CI jobs.

**Comment by rkothiya** [comment]
Now I have changed the jobs/build-rpms.yml also.

**Review by nixpanic — APPROVED** [review]

**Pull request merged**